be/src/exec/schema_scanner/schema_loads_scanner.cpp

```diff
@@ -30,6 +30,10 @@ SchemaScanner::ColumnDesc SchemaLoadsScanner::_s_tbls_columns[] = {
         {"PROGRESS", TYPE_VARCHAR, sizeof(StringValue), false},
         {"TYPE", TYPE_VARCHAR, sizeof(StringValue), false},
         {"PRIORITY", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"SCAN_ROWS", TYPE_BIGINT, sizeof(int64_t), false},
+        {"FILTERED_ROWS", TYPE_BIGINT, sizeof(int64_t), false},
```

Comment:
1. What's the difference between `FILTERED_ROWS` and `UNSELECTED_ROWS`?
2. How to understand the `UNSELECTED_ROWS`?